lib/streamlit/session_state.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2018-2021 Streamlit Inc.
```

Comment:
I think this file can't be called `session_state` for the desired API to work. The module conflicts with having an object of the same name in the `streamlit` module namespace.

Reply:
Ah, yeah. I'm not sure how I managed to miss that this name would result in a name conflict. Added a new `state` directory so this file is now named `lib/streamlit/state/session_state.py`.

Reply:
I didn't realize it would be an issue until I was staring python's confused error messages in the face when I created the current API in the prototype. The state dir was an excellent idea, much nicer than my hack of renaming it `session`.